packages/salesforcedx-vscode-apex/src/views/testRunner.ts

```diff
@@ -77,7 +80,20 @@ export class ApexTestRunner {
   }
 
   public getTempFolder(): string {
-    return os.tmpdir();
+    if (vscode.workspace && vscode.workspace.workspaceFolders) {
+      const sfdxDir = path.join(
+        vscode.workspace.workspaceFolders[0].uri.fsPath,
+        '.sfdx'
+      );
+      const apexTestPath = path.join(sfdxDir, 'apexTests');
```

Comment:
I think we should change the path to be `.sfdx/tools/testresults/apex`. The reason for this is 1) I want to follow the current pattern we have for debug logs where we store them under `.sfdx/tools/` and 2) we might need to store test results that are not specific to apex e.g `aura`.

Reply:
Okay sounds good, I'll make that change!